prelegenci/index.html

```diff
@@ -177,6 +177,28 @@ <h3><a href="/prelegenci/{{ fileName }}/">{{ speaker.name }}</a></h3>
 
 					<div class="row">
 
+						<div class="col-sm-6">
```

Comment:
@Gutek @ptrstpp950 
Do I have to still add this section in the main 'prelegenci' site? I see that lines 144..176 builds this automatically but I'm not sure if it works.

Reply:
Let's try. I don't remember to be fair :)